Duvet/SheetViewController.swift

```diff
@@ -94,7 +94,7 @@ public class SheetViewController: UIViewController {
                                        name: UIResponder.keyboardWillChangeFrameNotification,
                                        object: nil)
 
-        if let sheetItem = sheetItems.last {
+        if let sheetItem = sheetItems.last, sheetItem.viewController.parent != self {
```

Comment:
Oh that's clever! Not a property I've thought to check. Maybe an inline comment would be good here to explain this check? It's odd that this works since `self.sheetView = sheetView` is invoked just a few lines lower in in the same method that `addChild(sheetItem.viewController)` is called. 🤔